src/components/UploadDropzone.tsx

```diff
@@ -201,18 +217,20 @@ export const UploadDropzone = ({
           </aside>
         </section>
       )}
-      <ButtonsWrapper>
-        <Button label="cancel" onClick={onCancel} override={{ marginRight: 16 }} />
-        <Button
-          disabled={acceptedFileItems.length === 0}
-          label="save"
-          type={ButtonType.Primary}
-          onClick={() => {
-            onSubmit()
-            onCancel()
-          }}
-        />
-      </ButtonsWrapper>
+      {onSubmit && (
```

Comment:
This block is in my opinion does not belong here. Those are a general form handling actions and should be handled there.